survey-data-dictionary.md

```diff
@@ -17,12 +55,16 @@ what each question can have as an answer.
 
 ### Are you already working as a software developer?
 
+- **Question Type**: Single Answer
+- **Variable**: `IsSoftwareDev`
 - **Type**: Boolean
 - **Options**: Yes (1), No (0)
 
 
```

Comment:
We should add info about the conditions when the question is shown.
For example, questions:
>Would you prefer to...
>Which one of these roles are you most interested in?
>When do you plan to start applying for developer jobs?
>About how much money do you expect to earn per year at your first developer job (in US Dollars)?
>Would you prefer to work...
>Are you willing to relocate for a job?

They will show up only if respondent answered "No" to the question:
>Are you already working as a software developer?

That's important for analysis to avoid some miss-leading facts.

And also there are other questions with dependent sub-questions like bootcamps, current job, mortgage etc.